pages/search/index.vue

```diff
@@ -264,6 +264,10 @@
           view: this.view
         };
 
+        if (current.reusability === null) {
```

Comment:
Let's combine this with the following, which should future-proof for other parameters:
```js
      updateCurrentSearchQuery(updates) {
        const current = {
          page: this.page || '1',
          qf: this.qfForSelectedFacets,
          query: this.query || '',
          reusability: this.reusability,
          view: this.view
        };

        const updated = { ...current, ...updates };

        // If any values are now `null`, remove them from the query
        for (const key in updated) {
          if (updated[key] === null) {
            delete updated[key];
          }
        }

        return updated;
      }
```